post-merge

```diff
@@ -1,3 +1,34 @@
 #!/bin/sh
-composer install --no-dev --no-suggest
+NO_COLOR="\033[0m"
+RED="\033[38;5;009m"
+GREEN="\033[38;5;010m"
+YELLOW="\033[38;5;011m"
+ORANGE="\033[38;5;214m"
+LIGHTPURPLE="\033[38;5;177m"
+PURPLE="\033[38;5;135m"
+CYAN="\033[38;5;014m"
+LIGHTBLUE="\033[38;5;39m"
+BLUE="\033[38;5;75m"
+DARKBLUE="\033[38;5;33m"
+LIGHTGRAY="\033[38;5;252m"
+DARKGRAY="\033[38;5;242m"
+BRIGHTRED="\033[91m"
+BOLD="\033[1m"
+
+echo "\n${GREEN}post merge hook start${NO_COLOR}\n"
```

Comment:
We could add a test for an untracked file e.g. `NO_AUTO_COMPOSER` here to disable this functionality and report "skipped" instead of "start".